pkg/tsdb/loki/parse_response.go

```diff
@@ -106,9 +114,68 @@ func lokiStreamsToDataFrames(streams loghttp.Streams, query *lokiQuery) data.Fra
 		valueField := data.NewField("", tags, values)
 
 		frame := data.NewFrame("", timeField, valueField)
+		frame.SetMeta(&data.FrameMeta{
+			Stats: extraStats,
```

Comment:
nit: can we just call these stats, instead of extraStats :)

Reply:
sure, done !:)